config/config.go

```diff
@@ -41,27 +39,28 @@ const (
 	SSLCertPathKey                      = "SSL_CERT"
 	SSLKeyPathKey                       = "SSL_KEY"
 	WireEncodingKey                     = "WIRE_ENCODING"
-	ReplicaGroupIDKey                   = "REPLICA_GROUP_ID" // TODO: read replica group id from IPFS metadata
-	PaymentExpirationThresholdBlocksKey = "PAYMENT_EXPIRATION_THRESHOLD_BLOCKS"
 	PaymentChannelStorageTypeKey        = "PAYMENT_CHANNEL_STORAGE_TYPE"
 	PaymentChannelStorageClientKey      = "PAYMENT_CHANNEL_STORAGE_CLIENT"
 	PaymentChannelStorageServerKey      = "PAYMENT_CHANNEL_STORAGE_SERVER"
 
 	defaultConfigJson string = `
 {
-	"auto_ssl_cache_dir": ".certs",
 	"auto_ssl_domain": "",
 	"blockchain_enabled": true,
 	"daemon_listening_port": 5000,
 	"daemon_type": "grpc",
+    "daemon_end_point": "http://localhost:8080",
```

Comment:
Doesn't correspond to daemon_listening_port two lines above.

BTW, at the moment we have two properties to configure essentially same thing: ```daemon_listening_port``` and ```daemon_end_point```. I can imaging setup where these two things are not the same but it can be surprising for users. Don't have idea how to fix it properly so far. One of the options don't support such setups.

Reply:
Opened issue https://github.com/singnet/snet-daemon/issues/141  to discuss it